lib/workers/repository/update/branch/get-updated.ts

```diff
@@ -474,7 +474,14 @@ async function managerUpdateArtifacts(
 ): Promise<UpdateArtifactsResult[] | null> {
   const updateArtifacts = get(manager, 'updateArtifacts');
   if (updateArtifacts) {
-    return await updateArtifacts(updateArtifact);
+    if (updateArtifact.config.skipArtifactsUpdate) {
```

Comment:
This will be easier to read.
```ts
  if (!updateArtifacts) {
    return null;
  }

  if (updateArtifact.config.skipArtifactsUpdate) {
    logger.debug(
      { manager, packageFileName: updateArtifact.packageFileName },
      'Skipping artifact update',
    );
    return null;
  }

  return await updateArtifacts(updateArtifact);
  ```